python/pip_install/extract_wheels/lib/bazel.py

```diff
@@ -97,8 +97,12 @@ def generate_copy_commands(src, dest, is_executable=False) -> str:
         str: A `copy_file` instantiation.
     """
     return textwrap.dedent(
+        # Here we use the aliased rule `rules_python_copy_file` to avoid
```

Comment:
If this is in a generated file, when does this conflict happen? I think this makes it more confusing tbh about what this rule is, the load site (`rules_python`) makes it pretty clear that this is vendored, and not via `bazel_skylib`?

Reply:
It wouldn't "conflict". If users write their own "copy_file" rule and load it in this file, I'd like to make the vendored rule more obvious since. This is should have no change in behavior but was a minor thing that came up in a project of mine where a new Bazel user got confused.